packages/wallet-manager/src/storages/memory-storage.ts

```diff
@@ -5,19 +5,19 @@ class MemoryStorage implements StorageAbstract {
   storage = new Map<string, any>();
 
   async getItem<T>(key: string): Promise<T | null> {
-    return this.storage.get(key);
+    return Promise.resolve(this.storage.get(key));
   }
 
   async setItem(key: string, value: string) {
-    this.storage.set(key, value);
+    await Promise.resolve(this.storage.set(key, value));
   }
 
   async removeItem(key: string) {
-    this.storage.delete(key);
+    await Promise.resolve(this.storage.delete(key));
   }
 
   async clear() {
-    return this.storage.clear();
+    return Promise.resolve(this.storage.clear());
```

Comment:
All these methods seem to deal with the `storage` variable, an instance of a standard javascript `Map` class. I don't think any of these can become `async` in the future, can they?

Reply:
@camsjams Since we are using the [Eslint rule](https://typescript-eslint.io/rules/require-await/) `require-await`, we need to make all `async` functions either use `await` or return a `Promise.resolve(...)` to satisfy the lint error.

Reply:
Or remove the `async` directive from the method, which in this case, is what might be ideal.